proof-market/application-developers/tutorials/writing-new-circuit.mdx

```diff
@@ -0,0 +1,111 @@
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+
+# Writing a new circuit
```

Comment:
Personally i feel that we can only have a 1 tutorial how to generate statement attributes:

1. First we should explain everything which is already explained in this file
2. Later we should explain the attributes of statement object

```
{
    "name": "arithmetic1",
    "description": "simple arithmetic1",
    "url": "empty",
    "inputDescription": "arith",
    "type": "placeholder-zkllvm",
    "isPrivate": false,
    "definition": {
        "verificationKey": "verification_key",
        "provingKey": "; ModuleID = 'llvm-link'\r\nsource_filename = \"llvm-link\"\r\ntarget datalayout = \"e-m:e-p:64:8-a:8-i16:8-i32:8-i64:8-v768:8-v1152:8-v1536:8\"\r\ntarget triple = \"assigner\"\r\n\r\n$__clang_call_terminate = comdat..."
    }
}
```
So give an example of one where we explain the fields. And then mention how we can fetch and post stamenets...

WDYT?